### DISCUSSION TITLE
Curso Fullstack .NET - Testar sem banco de dados

### DISCUSSION THREAD

**Discussion by thiagovargas** [balta-io/forum#581] [Tira Dúvidas] [answered]
Pessoal, estou seguindo o curso Fullstack .NET assistindo e implementando, mas, não pretendo usar EntityFramework (quero ver se implemento com Dapper), mas não queria usar banco de dados nesse momento. 
É possível eu testar minha API de outra forma?
Queria tentar fazer uma implementação igual o Balta fez no curso de dominios ricos onde criava todas as regras que forem necessárias no dominio e criar testes para cada um deles.
Então... é possível eu fazer esses domínios assim e testar minha API sem banco de dados?
Me dêem uma orientação, por favor.
Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @thiagovargas, como vai?

Vi que fechou o tópico... caso precise de alguma ajuda, é só reabrir e me marcar no comentário!

**Reply by thiagovargas** [reply]
Fala Balta, eu acabei fechando esse pq na sua plataforma eu estou cadastrado com o email aqui da empresa, ai quando criei a duvida aqui com meu usuario pessoal, achei que não pudesse. Mas voce já me respondeu no outro topico que abri, muito obrigado
